scripts/claude-review-companion.mjs

```
@@ -328,7 +328,27 @@ function printUsage() {
 
-const BRIDGE_COMMAND_USAGE = Object.freeze({
+// This registry is intentionally consulted before any command-specific parser or
+// handler. Help must be a pure read of static CLI metadata: it cannot snapshot a
+// workspace, probe credentials, create state, or start a provider/runtime.
+const COMMAND_USAGE = Object.freeze({
+  enable: ["codex-claude enable [--config <path>] [--dry-run] [--json]", "Register the local Codex plugin without changing configuration when --dry-run is set."],
+  setup: ["codex-claude setup [--cwd <dir>] [--json]", "Report local helper setup and Claude authentication readiness."],
+  doctor: ["codex-claude doctor [--cwd <dir>] [--config <path>] [--job-dir <dir>] [--json] [--probe-runtime]", "Inspect local runtime readiness. --probe-runtime performs a live Claude probe; help never does."],
+  folder: ["codex-claude folder <path> [review flags] [focus text]", "Review a directory through the review lane."],
+  review: ["codex-claude review [review flags] [focus text]", "Run the standard read-only review lane."],
+  version: ["codex-claude version", "Print the installed companion version."],
```

Biome (2.5.7)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)